evm-ds/src/main.rs

```diff
@@ -222,7 +228,10 @@ async fn run_evm_impl(
                 result.set_trace(listener.traces.into_iter().map(Into::into).collect());
                 result.set_logs(logs.into_iter().map(Into::into).collect());
                 result.set_remaining_gas(remaining_gas);
-                debug!("Result: {:?}", result);
+                info!(
```

Comment:
Is `info` deliberate? the rest are `debug`

Reply:
yes, deliberate, so that we have all relevant info in one line and no other noise.